Sources/Build/llbuild.swift

```diff
@@ -241,7 +241,7 @@ public struct LLBuildManifestGenerator {
                 //FIXME: Should we add build time dependency on dependent targets?
                 inputs: [path.source.asString],
                 outputs: [path.object.asString],
-                args: [try plan.buildParameters.toolchain.getClangCompiler().asString] + args,
+                args: [try plan.buildParameters.toolchain.getTool(toolName: "clang").asString] + args,
```

Comment:
I think we should still have a `getClangCompiler()` method in `UserToolchain` which calls `getTool(toolName: "clang")` just so that we don't have the toolname strings at all callsites. Same for the other uses and then we can make `getTool(toolName:)` private.

Reply:
done